lib/config/validation.spec.ts

```diff
@@ -920,5 +921,56 @@ describe('config/validation', () => {
         },
       ]);
     });
+
+    it('errors if unallowed header in hostRules', async () => {
```

Comment:
```suggestion
    it('errors if forbidden header in hostRules', async () => {
```

Or maybe `blocked` is better. At least drop `unallowed`.